will/plugins/productivity/images.py

```diff
@@ -9,19 +10,29 @@ class ImagesPlugin(WillPlugin):
     @respond_to("image me (?P<search_query>.*)$")
     def image_me(self, message, search_query):
         """image me ___ : Search google images for ___, and post a random one."""
+
+        if not (getattr(settings, "GOOGLE_API_KEY", False) and getattr("GOOGLE_CUSTOM_SEARCH_KEY")):
```

Comment:
That means that this line will die with an AttributeError if the setting is not defined at all.

    class example(objects):
        real_attr = True

    getattr(example, real_attr)  # Returns True, as you'd expect
    getattr(example, fake_attr, False)  # Returns False
    getattr(example, fake_attr)  # Dies with AttributeError

However you have brought my attention to the second half of that line that will die in this exact way - so I'll fix that up.